README.md

```diff
@@ -75,30 +75,39 @@ The Enterprise documentation uses paid algolia indices, which auto-update every
 
 ## Generating the Admin API, CLI and Configuration Documentation
 
+The automated docs currently require using both the kong/kong repo and this repo (kong/docs.konghq.com, kong/docs for short) in combination.
+You also will need to know the <release> you are trying to generate docs for. The release looks
+like `0.14.x` in kong/docs, and like `0.14.1` or `0.14.2` in kong/kong.
+
 - Make sure that the `resty` and `luajit` executables are in your `$PATH` (installing kong should install them).
 - Several Lua rocks are needed. The easiest way to get them all is to execute `make dev` in the Kong folder
 - Have a local clone of Kong.
-- In the Kong repository, check out the desired branch/tag/release.
+- In the kong/kong repository, check out the desired branch/tag/release.
+
 - To generate the Admin API docs:
-  - Run: `KONG_PATH=path/to/your/kong/folder KONG_VERSION=0.14.x gulp admin-api-docs`
-  - This command will attempt to:
-    * Compare Kong's schemas and Admin API routes with the contents of the file
-      `autodoc-admin-api/data.lua` and error out if there's any mismatches or missing data.
-    * If no errors were found, a new `admin-api.md` file will be generated in the path corresponding
-      to the provided KONG_VERSION.
+  - On the kong/kong repo, run `./scripts/autodoc-admin-api`
+  - Copy the output from `kong/kong/autodoc/output/admin-api/` into `kong/docs/app/0.14.x` (replace with current release)
+  - Copy `kong/kong/autodoc/output/nav/docs_nav.yml.admin-api.in` into `kong/docs/autodoc-nav/docs_nav_0.14.x.yml.head.in`. Replace 0.14.x with release.
+  - On the kong/docs repo, run `KONG_VERSION=0.14.x luajit ./autodoc-nav/run.lua` (replace `0.14.x` with current release). This will merge the navigation
+    info in the right place.
+  - These commands generate a big file for the admin API and a smaller file for the navigation, which needs to be inserted in a
+  - specific place in the navigation yaml file.
```

Comment:
@kikito The smaller file is used as input by the `autodoc-nav` script. And it also generates another big Admin API file specific for DB-less.